tedana/utils/utils.py

```diff
@@ -133,17 +133,19 @@ def load_data(data, n_echos=None):
                              '{}'.format(data))
         else:  # individual echo files were provided (surface or volumetric)
             fdata = np.stack([load_image(f) for f in data], axis=1)
-            return np.atleast_3d(fdata), data[0]
+            fdata = np.atleast_3d(fdata)
+            ref_img = check_niimg(data[0])
+    else:
```

Comment:
I don't think this will work as-is for z-concatenated images. The `if isinstance(data, list)` will _always_ be triggered because of argparse, so the `else` clause here never will. I think we can get rid of this whole `if`-`else` block and move the code below this `else` section up under `if len(data) == 1`.